apps/tasks/messages.js

```diff
@@ -29,6 +35,28 @@ module.exports = function (grunt) {
         }
       });
     });
+
+    /**
+     * We want to make sure that we aren't missing any locale files in any
+     * namespaces; specifically, if we're missing locale information in any
+     * context, we want to default to English rather than landing in an error
+     * state.
+     *
+     * In this case, after building out all files found by examining apps/i18n
+     * we want to go through all combinations of locales and namespaces we
+     * encountered. Any combinations that are missing a built file should
+     * simply clone said file from english.
+     */
+    const {dest} = this.options();
+    locales.forEach(function (locale) {
+      namespaces.forEach(function (namespace) {
+        const expected = path.join(dest, locale, namespace + '.js');
+        if (!grunt.file.exists(expected)) {
+          const fallback = expected.replace(locale, 'en_us');
+          grunt.file.copy(fallback, expected);
```

Comment:
Is it worth outputting a warning or something if we get into this situation?

Reply:
oh, that seems valuable. I'm not sure of the best way to do that from the build, but I'll investigate

Reply:
gonna go ahead and merge this as-in, and pursue this idea in a separate PR